src/main/java/com/jitterted/mobreg/adapter/in/web/member/MemberController.java

```diff
@@ -38,7 +38,8 @@ public String showEnsemblesForUser(Model model, @AuthenticationPrincipal Authent
         model.addAttribute("register", memberRegisterForm);
 
         List<Ensemble> ensembles = ensembleService.allEnsemblesByDateTimeDescending();
-        List<EnsembleSummaryView> ensembleSummaryViews = EnsembleSummaryView.from(ensembles, member.getId());
+        List<Member> allUsers = memberLookup.findAll();
```

Comment:
Use MemberService directly here, not through MemberLookup (see comment below).

Reply:
Done